collectors/deployments_collector.go

```diff
@@ -51,6 +52,21 @@ func NewDeploymentsCollector(
 		[]string{"bosh_deployment", "bosh_stemcell_name", "bosh_stemcell_version", "bosh_stemcell_os_name"},
 	)
 
+	deploymentVMInfoMetric := prometheus.NewGaugeVec(
```

Comment:
Please use the same name for this var as the one defined at the `DeploymentsCollector` struct